Fix/resource busy rmdir

viralgupta opened the pull request:
fixed `Error: EBUSY: resource busy or locked, rmdir '../node'` error while creating a neu app on Windows. 
This would leave a folder without removing the .tmp files from the folder.

Previous error
![image](https://github.com/neutralinojs/neutralinojs-cli/assets/119971154/28bbab9a-b2c8-47d3-b37d-cd6aaca2aecc)

Current Solution
![image](https://github.com/neutralinojs/neutralinojs-cli/assets/119971154/e638ca2a-9664-44b1-893d-ee4c92086194)

shalithasuranga commented:
Hello @viralgupta Thanks so much for sending this pull request :) Could you please let us know how to reproduce this issue?

Thanks :tada:

viralgupta commented:
Hi @shalithasuranga , this issue would occur if the default branch were something other than "main," in my case, it was the "master" branch for the "viralgupta/node-neutralinojs-template" template. I had renamed the branch after seeing this issue. So, to reproduce the issue, I have changed it back to the "master" branch for now.

shalithasuranga commented:
Closing this issue since it doesn't occur with most templates and there are no issues similar reported by other developers. Thanks so much :tada:
